pkg/interface/link/src/js/components/settings.js

```diff
@@ -0,0 +1,301 @@
+                  api.setSpinner(true);
+                  api.metadataAdd(
+                    props.resourcePath,
+                    props.groupPath,
+                    props.resource.title,
+                    props.resource.description,
+                    props.resource['date-created'],
+                    state.color
+                  ).then(() => {
+                    api.setSpinner(false);
+                    this.refs.color.innerText = "Saved";
+                  });
+                }
+              }}>
+              Save
+            </span>
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  render() {
+    const { props, state } = this;
+    const isinPopout = this.props.popout ? "popout/" : "";
+
+    let writeGroup = Array.from(props.group.values());
+
+    if (props.groupPath === undefined) {
+      return <LoadingScreen/>;
+    }
+
+    if (!!state.isLoading) {
+      return (
+        <div className="h-100 w-100 overflow-x-hidden flex flex-column white-d">
+          <div
+            className="w-100 dn-m dn-l dn-xl inter pt4 pb6 pl3 f8"
+            style={{ height: "1rem" }}>
+            <Link to="/~link">{"⟵ All Collections"}</Link>
+          </div>
+          <div
+            className="pl4 pt2 bb b--gray4 b--gray2-d bg-gray0-d flex relative overflow-x-scroll overflow-x-auto-l overflow-x-auto-xl flex-shrink-0"
+            style={{ height: 48 }}>
+            <SidebarSwitcher
+              sidebarShown={this.props.sidebarShown}
+              popout={this.props.popout}
+            />
+            <Link to={makeRoutePath(props.resourcePath, props.popout)}
+            className="pt2 white-d">
+              <h2
+                className="mono dib f9 fw4 v-top"
+                style={{ width: "max-content" }}>
+                {props.resource.title}
+              </h2>
+            </Link>
+            <LinksTabBar {...props}/>
+          </div>
+          <div className="w-100 pl3 mt4 cf">
+            <h2 className="f8 pb2">Removing...</h2>
+          </div>
+        </div>
+      );
+    }
+
+    return (
+      <div className="h-100 w-100 overflow-x-hidden flex flex-column white-d">
+        <div
+          className="w-100 dn-m dn-l dn-xl inter pt4 pb6 pl3 f8"
+          style={{ height: "1rem" }}>
+          <Link to="/~chat/">{"⟵ All Chats"}</Link>
+        </div>
+        <div
+          className="pl4 pt2 bb b--gray4 b--gray1-d flex relative overflow-x-scroll overflow-x-auto-l overflow-x-auto-xl flex-shrink-0"
+          style={{ height: 48 }}>
+          <SidebarSwitcher
+            sidebarShown={this.props.sidebarShown}
+            popout={this.props.popout}
+          />
+          <Link to={makeRoutePath(props.resourcePath, props.popout)}
+          className="pt2">
+            <h2
+              className="mono dib f9 fw4 v-top"
+              style={{ width: "max-content" }}>
+              {props.resource.title}
```

Comment:
One comment for all four views: using `mono` on user-dictated name.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/20846414/76009267-f090e600-5ede-11ea-9a86-6e5fef813163.png">

So, is it cool and good to put the user's dictated name in the header? I don't know, maybe? But we don't do that in Chat, we put the endpoint in the header. Either: 
1. this has to change to Inter and we add user dictated names to the header in Chat or
2. This displays the endpoint name to match Chat

@urcades makes the call.

Reply:
ftr, displaying the endpoint name will give you `/it-s-a-collection-4897` or similar.

Reply:
Option 1 is preferred.

Very aware we want to retain the context for where a chat "lives" with respect to its host ship, so we'll need to think about how to obviously surface this info at some point, whether the endpoint is displayed onHover or below the name in a newly structured header or otherwise.

Filing this under something I need to think about for OS1+ updates.

Reply:
Then I'm adding option 1 to internal kanban and this is resolved — just remove `mono` @Fang-